src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/common/MapsPageBase.java

```diff
@@ -9,4 +9,19 @@ public MapsPageBase(WebDriver driver) {
         super(driver);
     }
 
+    public boolean isZoomInBtnPresent() {
+        return false;
+    }
+
+    public boolean isZoomOutBtnPresent() {
+        return false;
+    }
+
+    public String zoomInCoord() {
+        return " ";
+    }
+
+    public String zoomOutCoord() {
+        return " ";
+    }
```

Comment:
MapsPageBase should be abstract and make all methods abstract too
And create MapsPage for iOS because it isn't there

Reply:
done